android bug latest version

<img width="1095" alt="Снимок экрана 2020-12-10 в 14 17 33" src="https://user-images.githubusercontent.com/23016311/101771400-8acb2d80-3af2-11eb-9eb2-9327adeaca06.png">

After update from "react-native-screens": "2.15.0" to "react-native-screens": "2.16.0", android not bundled

I have this problem too:

`..\node_modules\react-native-screens\android\src\main\java\com\swmansion\rnscreens\ScreenStack.java:189: error: cannot find symbol
            getOrCreateTransaction().setCustomAnimations(R.anim.rns_slide_in_from_right, R.anim.rns_slide_out_to_left);   
                                                               ^
  symbol:   variable rns_slide_in_from_right
  location: class anim`

--- a/package.json
+++ b/package.json
@@ -24,6 +24,7 @@
     "createNativeStackNavigator/",
     "android/src/main/AndroidManifest.xml",
     "android/src/main/java/",
+    "android/src/main/res",
     "android/build.gradle",
     "ios/",
     "RNScreens.podspec",

